steps/browserstack-bitrise-espresso-step/step-info.yml

```
@@ -0,0 +1 @@
+maintainer: Verified
```

```suggestion
maintainer: verified
```

This is the correct value I think